cypress/integration/page.spec.js

```diff
@@ -64,509 +64,509 @@ describe('', () => {
-    });
-
-    it('should have people table', () => {
-      page.peopleTable().should('exist');
-    });
-
-    it('should have one selected person', () => {
-      page.people().eq(1)
-        .should('have.class', SELECTED_PERSON_CLASS);
-
-      cy.get('[data-cy="person"].' + SELECTED_PERSON_CLASS)
-        .should('have.length', 1);
-    });
-
-    it('should highlight person with given slug', () => {
-      page.visit('/#/people/jan-van-brussel-1714');
-
-      page.people().eq(3)
-        .should('have.class', SELECTED_PERSON_CLASS)
-    });
-
-    it('should allow to select another person', () => {
-      page.visit('/#/people/jan-van-brussel-1714');
-
-      page.people().eq(3)
-        .find('td').eq(0)
-        .find('a')
-        .click();
-
-      page.assertHash('#/people/jan-van-brussel-1714')
-
-      page.people().eq(3)
-        .should('have.class', SELECTED_PERSON_CLASS);
-
-      page.people().eq(1)
-        .should('not.have.class', SELECTED_PERSON_CLASS);
-    });
-  });
-
-  describe('#/people/:wrong-slug page', () => {
-    beforeEach(() => {
-      page.mockPeople();
-      page.visit('/#/people/non-existing-slug');
-    });
-
-    it('should have correct address', () => {
-      page.assertHash('#/people/non-existing-slug');
-    });
-
-    it('should have People nav link active', () => {
-      page.nav()
-        .contains('a', 'People')
-        .and('have.class', ACTIVE_NAV_LINK_CLASS);
-    });
-
-    it('should have only correct title', () => {
-      page.assetTitle('People Page');
-    });
-
-    it('should have people table', () => {
-      page.peopleTable().should('exist');
-    });
-
-    it('should not have a selected person', () => {
-      cy.get('[data-cy="person"].' + SELECTED_PERSON_CLASS)
-        .should('not.exist');
-    });
-
-    it('should allow to select a person', () => {
-      page.people().eq(1)
-        .find('td').eq(0)
-        .find('a')
-        .click();
-
-      page.assertHash('#/people/emma-de-milliano-1876')
-
-      page.people().eq(1)
-        .should('have.class', SELECTED_PERSON_CLASS);
-    });
+  //   it('should have navigation', () => {
+  //     page.mockPeople();
+  //     page.visit('/');
+  //     page.nav().should('exist');
+  //   });
+
+  //   it('should mark Home nav link as active', () => {
+  //     page.mockPeople();
+  //     page.visit('/');
+  //     page.nav()
+  //       .contains('a', 'Home')
+  //       .should('have.attr', 'href', '#/')
+  //       .and('have.class', ACTIVE_NAV_LINK_CLASS);
+  //   });
+
+  //   it('should not mark People nav link as active', () => {
+  //     page.mockPeople();
+  //     page.visit('/');
+  //     page.nav()
+  //       .contains('a', 'People')
+  //       .should('have.attr', 'href', '#/people')
+  //       .and('not.have.class', ACTIVE_NAV_LINK_CLASS);
+  //   });
+
+  //   it('should have only correct title', () => {
+  //     page.mockPeople();
+  //     page.visit('/');
+  //     page.assetTitle('Home Page');
```

Comment:
The function `assetTitle` is likely a typo and should be `assertTitle` to correctly describe its purpose of asserting the page title.